contracts/core/extensions/CoreModuleInteraction.sol

```diff
@@ -36,6 +36,18 @@ contract CoreModuleInteraction is
     CoreState,
     ReentrancyGuard
 {
+    modifier onlyModule() {
```

Comment:
We've kind of eschewed the use of modifiers why start now? Does this help with bytesize? As far as I remember modifiers get compiled into the function anyway so this increases bytesize right?

Reply:
In this case, it became really un-DRY to have the same modifier code for 8 functions